cfme/utils/appliance/__init__.py

```diff
@@ -494,6 +494,8 @@ def configure(self, log_callback=None, **kwargs):
                 self.db.create_db_lvm()
             if on_openstack:
                 self.set_resolvable_hostname(log_callback=log_callback)
+            if db_address:
+                self.db_host = db_address
```

Comment:
Since you're now using `db_address` can you also update L441

Reply:
Agreed - I'd like to see a followup doing two things:

1. Update the docblock above
2. Improve the `db_host` and `self.db.address` setup, so that its not so circular, and operates as a property that isn't self referential.